hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -1267,9 +1275,7 @@ Actions:
 </div>
 
 <jsp:include page="footer.jsp" />
-<script src="/static/js/jquery.min.js" type="text/javascript"></script>
 <script src="/static/js/jquery.tablesorter.min.js" type="text/javascript"></script>
-<script src="/static/js/bootstrap.min.js" type="text/javascript"></script>
```

Comment:
Here I removed these JavaScript includes because `footer.jsp` is included just above and that includes these already.